app/src/lib/open-shell.ts

```diff
@@ -15,14 +15,20 @@ export function openShell(fullPath: string, shell?: string) {
     return spawn('START', [shell || 'cmd'], { shell: true, cwd: fullPath })
   }
 
+  if (__LINUX__) {
+    const commandArgs = ['--working-directory', fullPath]
+    return spawn('gnome-terminal', commandArgs, { shell: true })
```

Comment:
Thanks. Maybe something to put in a follow-up PR. Right now I'm running a pretty minimal Linux installation with only Xfce and awesome installed. No GNOME dependencies.